Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -28,7 +29,10 @@ class LoadDataFixturesDoctrineCommand extends DoctrineCommand
     /** @var SymfonyFixturesLoader */
     private $fixturesLoader;
 
-    public function __construct(SymfonyFixturesLoader $fixturesLoader, ?ManagerRegistry $doctrine = null)
+    /** @var ContainerInterface */
+    private $purgerFactories;
+
+    public function __construct(SymfonyFixturesLoader $fixturesLoader, ContainerInterface $purgerFactories, ?ManagerRegistry $doctrine = null)
```

Comment:
Please avoid the BC break

Reply:
Default factories to null, current behavior if null?